app/Filament/Widgets/ContactGrowthChart.php

```diff
@@ -70,7 +70,7 @@ protected function getOptions(): array
 
     protected function getData(): array
     {
-        $runningTotalPerMonth = Cache::tags(['contacts'])
+        $runningTotalPerMonth = Cache::tags(['{{contacts}}'])
```

Comment:
This should only have one set of `{}` around this.

In the example you are copying from, it only has the double `{}` because it is within a string with a variable in it and String interpolation.